packages/components/psammead-sitewide-links/src/List/index.jsx

```diff
@@ -73,17 +73,13 @@ const StyledList = styled.ul`
     padding: ${GEL_SPACING} 0;
     margin-bottom: ${GEL_SPACING};
     grid-column: 1/-1;
-    @supports not (${grid}) {
-      width: 100%;
-    }
+    width: 100%;
   }
 `;
 
 const StyledListItem = styled.li`
+  display: inline-block;
   min-width: 50%;
```

Comment:
Same here:
```
@supports (${grid}) {
    display: block;
 }
```

Reply:
@OlgaLyubin is doing something similar in her PR https://github.com/bbc/psammead/blob/67db16d966d8122333c2cbb5dcebd278dc7f1995/packages/components/psammead-useful-links/src/index.jsx